index.ts

```diff
@@ -1,8 +1,10 @@
 import { IncomingMessage, ServerResponse } from 'http'
 
-export = function ienoopen () {
-  return function ienoopen (req: IncomingMessage, res: ServerResponse, next: () => void) {
+export function ienoopen () {
+  return function ienoopen (_req: IncomingMessage, res: ServerResponse, next: () => void) {
```

Comment:
Is the change from `req` to `_req` stylistic or does it serve some other purpose? If it's just stylistic, I'd prefer to stay consistent with the way most Express middleware is defined and use `req`.

This comment also applies to the change in the test file.

Reply:
Since we turned on `noUnusedParameters` in `tsconfig.json`, TS will complain about unused parameters here. Prefacing the parameter with an underscore makes the warning go away.